build-system/compile/single-pass.js

```diff
@@ -582,6 +583,7 @@ exports.singlePassCompile = async function(entryModule, options) {
     .then(intermediateBundleConcat)
     .then(eliminateIntermediateBundles)
     .then(thirdPartyConcat)
+    .then(removeInvalidSourcemaps)
```

Comment:
Why not just remove these two lines? Isn't that easier than generating incorrect sourcemaps and then deleting them?

https://github.com/ampproject/amphtml/blob/9fe86647f98ac9cf7576cbbe671b409e9f46302f/build-system/compile/single-pass.js#L688
https://github.com/ampproject/amphtml/blob/9fe86647f98ac9cf7576cbbe671b409e9f46302f/build-system/compile/single-pass.js#L694

Reply:
Tried it, it works. Sent out #23145

Reply:
The next step is for me to fix the sourcemaps, and it's easier to remove this line than adding all these back.

Reply:
Curious: what do lines 558-561 do? Would it be possible to roll them in to `compile()` so the files needn't be rewritten?

https://github.com/ampproject/amphtml/blob/4b86736870dc948e0c706fcac6317136d16af819/build-system/compile/single-pass.js#L548-L567

Reply:
I don't know about `wrapMainBinaries`. 

`intermediateBundleConcat` and `eliminateIntermediateBundles` could be combined, but `thirdPartyConcat` cannot.